src/get-options-overrides.ts

```diff
@@ -24,12 +24,6 @@ export function getOptionsOverrides({ useTsconfigDeclarationDir, cacheRoot, cwd
 		if (preParsedTsconfig.options.module === undefined)
 			overrides.module = tsModule.ModuleKind.ES2015;
 
-		const declaration = preParsedTsconfig.options.declaration;
-		if (!declaration)
-			overrides.declarationDir = undefined;
-		if (declaration && !useTsconfigDeclarationDir)
-			overrides.declarationDir = cwd;
-
```

Comment:
I removed this whole thing because `undefined` is the default ...but I think this should actually be:

```ts
        if (!useTsconfigDeclarationDir)	
			overrides.declarationDir = undefined;
```

So that it isn't set to `declarationDir` otherwise.